core/presolve.gms

```diff
@@ -112,15 +112,7 @@ loop(regi,
           );
     );
 
-  if (cm_emiscen eq 9 or (cm_emiscen eq 10),
```

Comment:
Sorry if I ask stupid questions, but for emiscen = 6, do we now have the right value here? Because we are in core/presolve, so the code in core/postsolve has not necessarily been executed, right? Or is it fine to just use the value saved in the last iteration?

Reply:
in the very first Nash iteration we will have the wrong value but afterwards it will use the number from the last iteration which is o.k. I would say

Reply:
fine!